code/modules/mining/ores_coins.dm

```diff
@@ -23,9 +23,13 @@
 	..()
 
 /obj/item/weapon/ore/Crossed(atom/movable/AM)
+	var/show_message = TRUE
+	for(var/obj/item/weapon/ore/O in loc)

```

Comment:
In testing, it only shows one message; the ores are cycled and removed from the loc before any more than one loop can go through them.